code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -327,10 +327,10 @@ Credit where due:
 	Here is the layout of Reebe, from inner to outter:\
 	<ul>\
 	<li><b>Ark Chamber:</b> Houses the Ark in the very center.</li>\
-	<li><b>Listening Station:</b> (Bottom Left Corner of circle) Contains intercoms, a telecomms relay, and a list of frequencies.</li>\
-	<li><b>Observation Room:</b> (Bottom Right Corner of circle) Contains six camera observers. These can be used to watch the station through its cameras, as well as to teleport down \
+	<li><b>Listening Station:</b> (Bottom Left Corner of Circle) Contains intercoms, a telecomms relay, and a list of frequencies.</li>\
+	<li><b>Observation Room:</b> (Bottom Right Corner of Circle) Contains six camera observers. These can be used to watch the station through its cameras, as well as to teleport down \
 	to most areas. To do this, use the Warp action while hovering over the tile you want to warp to.</li>\
-	<li><b>Infirmary:</b> (Uper Right Corner of circle) Contains sleepers and basic medical supplies for superficial wounds. The sleepers can consume Vitality to heal any occupants. \
+	<li><b>Infirmary:</b> (Uper Right Corner of Circle) Contains sleepers and basic medical supplies for superficial wounds. The sleepers can consume Vitality to heal any occupants. \
```

Comment:
Why?

Reply:
As i recall the old map it was capitalized. But becides that i just want to catch the eye of the person who reads it.